src/solids4FoamModels/numerics/vfvCellPoint/vfvmCellPointExtended.C

```diff
@@ -681,8 +682,9 @@ void Foam::vfvm::Sp
             
```

Comment:
You should check by hand if this coeff calculation is correct.
Also, it may be easiest to first code the small-strain version; in that case, I guess `pBarSensitivityI` should be equal to the bulk modulus.